docs/resources/integration_aws_account.md

```diff
@@ -0,0 +1,221 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_integration_aws_account Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog - Amazon Web Services integration resource. This can be used to create and manage Datadog - Amazon Web Services integration.
+---
+
+# datadog_integration_aws_account (Resource)
+
+Provides a Datadog - Amazon Web Services integration resource. This can be used to create and manage Datadog - Amazon Web Services integration.
```

Comment:
```suggestion
Provides a Datadog—Amazon Web Services integration resource. This can be used to create and manage Datadog—Amazon Web Services integration.
```